docs/guide.md

```diff
@@ -80,33 +82,54 @@ Make sure to check the [**Compatibility with other Fabric mods**](#compatibility
 9. Place the downloaded files (and optionally your other Fabric mods) into the mods folder. The location of the mods folder depends on  your operating system:
     - Windows: `%APPDATA%\.minecraft\mods`  
     - Linux: `~/.minecraft/mods`  
     - MacOS: `~/Library/Application Support/minecraft/mods`
 10. Next time when you open the Minecraft launcher, you should have the following profile (the version numbers may be different):  
     ![fabricprofile](guide-resources/fabricprofile.png)  
     Play the game with this profile selected to run the game with Iris Shaders and the rest of your Fabric mods!
-    
+
 ## Using Iris
 
 After you install Iris using one of the methods described in the [Installing Iris](#installing-iris) section, make sure that it is properly installed by going into the Minecraft video settings. The video settings menu should look like this:  
 
 ![videosettings](guide-resources/videosettings.png)
 
-You probably noticed that the menu is completely different. It is a reworked video settings menu provided by Sodium. In the **General** and **Quality** tab, you can find all the usual settings from vanilla Minecraft which you can change to fit your preferences. The settings in the Performance and Advanced tabs should not be modified in most cases.
+You probably noticed that the menu is completely different. It is a reworked video settings menu provided by Sodium. In the **General** and **Quality** tab, you can find all the usual settings from vanilla Minecraft which you can change to fit your preferences. The settings in the Performance and Advanced tabs should not be modified in most cases.  
 
 Shader packs are managed in the **Shader packs** tab. The shader pack menu should look like this:  
 
 ![shaderpacksmenu](guide-resources/shaderpacksmenu.png)
 
-This menu shows shader packs which are in the shader packs folder. You can add new shader packs by dragging and dropping them into this menu, or by clicking the "Open Shader Pack Folder..." button and adding the shader packs into the opened folder. To select a shader pack, click on "Shaders: Disabled" to enable them. After that, select the shader pack you want to enable by clicking on it, then enable the selected shader by clicking the Apply button. After setting up your shaders, click Done to go back to the video settings menu.
+This menu shows shader packs which are in the shader packs folder. You can add new shader packs by dragging and dropping them into this menu, or by clicking the "Open Shader Pack Folder..." button and adding the shader packs into the opened folder. After adding your shader packs to the shader pack folder, click the ![refreshbutton](guide-resources/refreshbutton.png) button to refresh the list of shaders. To select a shader pack, click on "Shaders: Disabled" to enable them. After that, select the shader pack you want to enable by clicking on it, then enable the selected shader by clicking the Apply button.  
```

Comment:
Alt text should be human readable - this should be "refresh" instead of "refreshbutton" so it flows inline with the text.

Reply:
I replaced the image of the button with a small description of where it is (top right). Hopefully this is sufficient